veterans/components/homePage/mapSection/initiativesFilter/InitiativesFilter.tsx

```
@@ -49,6 +49,36 @@ const CheckboxGroup: React.FC<
+
+    switch (e.key) {
+      case 'ArrowDown':
+        e.preventDefault();
+        const nextIndex = (index + 1) % totalCheckboxes;
+        document
+          .getElementById(`checkbox-${categories[nextIndex].id}`)
+          ?.focus();
+        break;
+      case 'ArrowUp':
+        e.preventDefault();
+        const prevIndex = (index - 1 + totalCheckboxes) % totalCheckboxes;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)